### PR TITLE
Definitions list command

### PR DESCRIPTION
**Pull request by Zifius**
Implements #10

### CODE REVIEW

File: handlers.py [hunk 1]
```diff
@@ -24,6 +24,19 @@ def handle(self, bot, update):
...
+
+    def __init__(self, definition_repo):
+        super().__init__()
+        self.definitionRepo = definition_repo
+
+    def handle(self, bot, update):
+        definitions = self.definitionRepo.findAll()
+        reply = update.message.reply_text
+        for definition in definitions:
+            reply("/wtf {}".format(definition))
```
Comment (Romanow88) [inline handlers.py]: formatted string should be in parentheses.

File: handlers.py [hunk 2]
```diff
@@ -82,6 +95,7 @@ def handle(self, bot, update):
 
 
 definitionHandler = DefinitionHandler(definitionRepo)
+listDefinitionsHandler = ListDefinitionsHandler(definitionRepo)
```
Comment (Romanow88) [inline handlers.py]: The functionality here can be added to the definitions handler (separate method).

File: handlers.py (hunk 1 above)
Comment (Zifius) [reply handlers.py]: Not sure what you mean, looks like it works as is

File: handlers.py (hunk 2 above)
Comment (Zifius) [reply handlers.py]: Thanks, refactored

**Pull request merged**